codeclimate-wrapper

```diff
@@ -18,32 +18,52 @@ EOF
   exit 1
 }
 
+docker_for_mac() {
+  test -S /var/tmp/docker.sock > /dev/null 2>&1
+}
+
+socket_exists() {
+  test -S $socket_path > /dev/null 2>&1
+}
+
+socket_path() {
```

Comment:
> Are we in agreement then that the few lines I've been pasting is the
answer? Just adjust $socket_path up top and leave the rest of the script
alone?

Yes, that's fine with me.